src/collectd.conf.in

```diff
@@ -596,8 +596,11 @@
 #</Plugin>
 
 #<Plugin hugepages>
-#    ReportPerNodeHP "true"
-#    ReportRootHP "true"
+#    ReportPerNodeHP  true
+#    ReportRootHP     true
+#    ValuesPages      true
+#    ValuesBytes      false
```

Comment:
would be good to report the bytes also by default... hugepages can be 2MB or 1G - so reporting the number on it's own might not make sense